hooks/pre-commit

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+from __future__ import print_function
+from future import standard_library
```

Comment:
```
from future import standard_library
standard_library.install_aliases()
```


mover standard_library.install_aliases() here so that `urllib.request` can get imported